src/Datastore/RAMDataStore.h

```diff
@@ -28,17 +28,96 @@
 
 class RAMDataStore : public DataStore {
  public:
+  /**
+   * @brief    Add a User to the internal List of registered Users
+   * @param    User The User class to add
+   * @retval   TResultOpt possibly containing Error Message
+   */
```

Comment:
the exact same comments already exist in the Base Class `DataStore` - they can be removed here.
because in the case of a change, the comment only needs to be adapted in one place.

Reply:
done